README.md

```diff
@@ -130,6 +130,7 @@ Communications (Website, etc.)|Every other week on Thursday at 13:30 ET|[Google
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1pAxBekk6BadHpK717TbhYtBMOlyrgWSvp3QeYDPwwvk/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/1mHNnlCmO0XKUu0xPnKko3GOh0mxVoxjRiN8xmHRImew/edit#)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
 Client Instrumentation|Every Wednesday at 8:30 AM PT|[Google Doc](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w/edit)|[Slack](https://cloud-native.slack.com/archives/C0239SYARD2)|
+Operator|TODO time|TODO google doc|TODO slack|
```

Comment:
```suggestion
Kubernetes Operator|TODO time|TODO google doc|TODO slack|
```

Reply:
Do we need to setup a meeting and google doc? 

For the channel I would prefer to use the collector channel to ensure there is collaboration and visibility between collector group.

Reply:
Here is the typical format we use for SIG/Workgroup meeting notes: https://docs.google.com/document/d/1zBNzh1dw9HpXUexTaURRS7n_NQSzqNJSuqNVw9GhIe4/edit#
This doc is publicly shared, feel free to use it.

Reply:
If you don't need a meeting, simply put NA there :)

Reply:
thanks for the links

I have updated the PR with `NA` for the meeting at the moment. There will be probably meetings per requests